bin/nrnpyenv.sh

```diff
@@ -132,20 +133,40 @@ if test "$z" = "Darwin" ; then
     unset p
     unset d
     unset l
-  else
+    nrnpylib_provenance="sysconfig LIBDIR"
+  fi
+  if test "$z" = "" ; then
+    z=$($p -c '
+try:
+  from neuron import h
+  s=h.libpython_path()
+  s = s if ".dylib" in s else ""
+  print(s)
+except:
+  print("")
```

Comment:
I assume this doesn't print to stdout but output is assigned to variable, correct?

Reply:
correct. It is the value of z

Reply:
correct. it gets assigned to the bash shell variable nrn_pylib